app/src/main/java/com/duckduckgo/app/autocomplete/api/AutoCompleteApi.kt

```diff
@@ -16,16 +16,12 @@
 
 package com.duckduckgo.app.autocomplete.api
 
-import com.duckduckgo.app.browser.omnibar.QueryUrlConverter
 import com.duckduckgo.app.global.UriString
 import io.reactivex.Observable
 import javax.inject.Inject
 
 
-open class AutoCompleteApi @Inject constructor(
-        private val autoCompleteService: AutoCompleteService,
-        private val queryUrlConverter: QueryUrlConverter
-) {
+open class AutoCompleteApi @Inject constructor(private val autoCompleteService: AutoCompleteService) {
```

Comment:
Is this an exogenous change?

Reply:
Use smaller words! 😆 

The change wasn't needed or caused by anything; just noticed this class had an unused property and including here as a general tidy-up.